wallet/reference/multichain-api.md

```diff
@@ -0,0 +1,396 @@
+---
+description: See the Multichain API reference.
+sidebar_custom_props:
+  flask_only: true
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Multichain API
+
+:::flaskOnly
+:::
+
+Dapps can call Multichain API [methods](#methods) to create and manage
+[CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) connections with MetaMask.
+The API also provides [events](#events) that wallets can send to dapps to notify them of onchain or connection changes.
+
+:::note See also
+- [About the Multichain API](../concepts/multichain-api.md)
+- [Interact with multiple networks](../how-to/manage-networks/use-multichain.md)
+:::
+
+## Methods
+
+### `wallet_createSession`
+
+Creates a multichain connection with a wallet, authorizing that wallet with the specified set of scopes
+and properties.
+This method is defined in [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md).
+
+#### Parameters
+
+- `optionalScopes`: `object` - (Optional) [CAIP-217](https://chainagnostic.org/CAIPs/caip-217) authorization scopes the wallet can support in order to be used with this dapp.
+  If scopes are specified, only the following properties are supported:
+  - `references`: `array` - (Optional) A list of references to specific blockchains for the namespace of this scope.
+    This property is mainly used when there would otherwise be duplicate scopes.
+  - `methods`: `array` - A list of JSON-RPC methods the wallet must support in order to be used with this dapp.
+  - `notifications`: `array` - A list of JSON-RPC notifications the wallet must support in order to be used with this dapp.
+  - `accounts`: `array` - (Optional) A list of [CAIP-10](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md)
+    account IDs valid within this scope.
+    Dapps should only supply this property when they know the set of accounts they would like the user to permit.
+    When supplied, these accounts are preselected by default in the account selection process.
+    Dapps typically omit this property for the user to select their own accounts.
+- `sessionProperties`: `object` - (Optional) Properties that the wallet can use to determine if the connection is valid.
+- `requiredScopes`: `object` - (Optional) [CAIP-217](https://chainagnostic.org/CAIPs/caip-217) authorization scopes the wallet must support in order to be used
+  with this dapp.
+  We don't recommend using `requiredScopes` with MetaMask.
```

Comment:
We treat all requiredScopes as optional despite the name

Reply:
Added to description.